python2_nbd_client.py

```diff
@@ -71,15 +117,15 @@ def _non_fixed_new_style_handshake(self, export_name):
         self._s.sendall(length)
         self._s.sendall(str.encode(export_name))
 
-        # non-fixed newstyle negotiation: we get this if the server is willing
+        # fixed newstyle negotiation: we get this if the server is willing
         # to allow the export
         buf = self._s.recv(8)
         self._size = struct.unpack(">Q", buf)[0]
         # ignore the transmission flags (& zeroes)
         self._s.recv(2 + 124)
 
     def _build_header(self, request_type, offset, length):
-        print "NBD request offset=%d length=%d" % (offset, length) 
+        print "NBD request offset=%d length=%d" % (offset, length)
```

Comment:
Did you mean to keep the prints?

Reply:
I left them just because they are handy for checking whether data is actually being read. I can remove though if you think it is too verbose